### PR TITLE
Minor updates to `README.md`

### PR DESCRIPTION
**Pull request by mary-cleaton**
## Description

Minor updates to `README.md` as follows:
 - Add contact email.
 - Add dedication.
 - Add logos for linter and formatter.
 - Edit pre-requisites section.
 - Update acknowledgements list.

## Type of change

- [ ] Bug fix - *non-breaking change*
- [ ] New feature - *non-breaking change*
- [ ] Breaking change - *backwards incompatible change, changes expected behaviour*
- [x] Non-user facing change, structural change, dev functionality, docs ...

## Checklist:

- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code appropriately, focusing on explaining my design decisions (explain why, not how).
- [x] I have made corresponding changes to the documentation (comments, docstring, etc.. )
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have updated the change log.

<br>

#  Peer review
Any new code includes all the following:

- **Documentation**: docstrings, comments have been added/ updated.
- **Style guidelines**: New code conforms to the project's contribution guidelines.
- **Functionality**: The code works as expected, handles expected edge cases and exceptions are handled appropriately.
- **Complexity**: The code is not overly complex, logic has been split into appropriately sized functions, etc..
- **Test coverage**: Unit tests cover essential functions for a reasonable range of inputs and conditions. Added and existing tests pass on my machine.

### Review comments
Suggestions should be tailored to the code that you are reviewing. Provide context.
Be critical and clear, but not mean. Ask questions and set actions.
<details><summary>These might include:</summary>

- bugs that need fixing (does it work as expected? and does it work with other code
  that it is likely to interact with?)
- alternative methods (could it be written more efficiently or with more clarity?)
- documentation improvements (does the documentation reflect how the code actually works?)
- additional tests that should be implemented
  - Do the tests effectively assure that it
  works correctly? Are there additional edge cases/ negative tests to be considered?
- code style improvements (could the code be written more clearly?)
</details>
<br>

*Further reading: [code review best practices](https://best-practice-and-impact.github.io/qa-of-code-guidance/peer_review.html)*

### CODE REVIEW

**Review by alani-odunlami — APPROVED** [review]
All look good.

File: README.md [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
 
-`scalelink` is a Python and PySpark codebase built with Python 3.8-3.10 and Spark 3.5.1.
+- Python 3.8-3.10
+- Spark 3.5.1
 
 ## Method
 
 The Scalelink method is a novel probabilistic data linkage method. It uses a scaling algorithm based on correspondence analysis. A key potential advantage is that it utilises linkage variable dependence. In contrast, the current gold-standard probabilistic data linkage method, the Fellegi-Sunter algorithm, assumes that linkage variables are independent, an assumption which is violated to at least some extent in all linkage problems. For example, forenames are correlated with middle names, surnames, age, gender, home address, etc.
 
 The Scalelink method is **experimental**. This implementation is has been open-sourced to facilitate further research regarding the suitability of the Scalelink method for real-world data linkage, particularly on Big Data. The authors **do not** currently endorse using the Scalelink method to produce linked datasets for research, analysis or statistics.
```
Comment (alani-odunlami) [inline README.md]: I think there is an error here:  'This implementation **is has** been open-sourced'

File: setup.cfg [hunk 2]
```diff
@@ -6,7 +6,7 @@ author_email = datalinkage@ons.gov.uk
 description = A Python and PySpark implementation of the Scalelink method of data linkage by Goldstein et al.
 long_description = file: README.md
 long_description_content_type = text/markdown
-url =
+url = https://github.com/ONSdigital/scalelink
```
Comment (alani-odunlami) [inline setup.cfg:9]: Link works perfectly.

File: README.md [hunk 3]
```diff
@@ -1,19 +1,29 @@
...
 
 This implementation has been created in response to recommendation six of the [Joined Up Data in Government: The Future of Data Linking Methods][guidance] cross-government review.
 
-The Data Linkage team within the Methodology and Quality Directorate of the Office for National Statistics (ONS) have authored this work. This team researches novel data linkage methods and applications to facilitate production of high-quality linked datasets for national statistics.
+This work is authored by the Data Linkage team within the Methodology and Quality Directorate (MQD) of the Office for National Statistics (ONS). This team researches novel data linkage methods and applications to facilitate production of high-quality linked datasets for national statistics.
+
+## Pre-requisites
 
-`scalelink` is a Python and PySpark codebase built with Python 3.8-3.10 and Spark 3.5.1.
+- Python 3.8-3.10
```
Comment (alani-odunlami) [inline README.md:14]: Is Python  3.11 not yet included?

Comment (mary-cleaton) [reply README.md:14]: Not yet. This is added in a separate PR.

File: README.md (hunk 1 above)
Comment (mary-cleaton) [reply README.md]: Corrected.

**Review by alani-odunlami — APPROVED** [review]
